locale/en/manager.po

```diff
@@ -2990,6 +2990,9 @@ msgstr "Issue identification, including volume, number, year, and title"
 msgid "emailTemplate.variable.issue.issuePublishedUrl"
 msgstr "The URL to the published issue"
 
+msgid "emailTemplate.variable.issue.issueTableOfContent"
+msgstr "The issue table of content such as list of all associated submissions."
```

Comment:
Does this need to go into the pkp-lib locale files? The rest of the feature is OJS-only, so the locale keys should stay there too.

Reply:
Right, moved to OJS app locale files .